lib/components/NavMenu.jsx

```diff
@@ -43,7 +43,7 @@ const NavMenu = ({ menuItems, selectedItemId, onItemSelected, cssClass }) => (
     <div className={cssClass}>
         {
             menuItems.map((item, index) => {
-                const hotkey = `alt+${index + 1}`;
+                const hotkey = `Alt+${index + 1}`;
```

Comment:
This might cause the hotkey to fail, since `mousetrap` only handles lowercase modifier IIRC. Please double-check.

Reply:
On the topic of hotkeys: does the dialog offer `Enter` and `Esc` hotkeys for the OK/Cancel buttons?